server/src/main/java/org/apache/druid/rpc/StandardRetryPolicy.java

```diff
@@ -62,11 +70,27 @@ public static Builder builder()
     return new Builder();
   }
 
+  /**
+   * Standard unlimited retry policy. Never stops retrying as long as errors remain retryable.
+   * See {@link ServiceClient} documentation for details on what errors are retryable.
+   */
   public static StandardRetryPolicy unlimited()
   {
     return DEFAULT_UNLIMITED_POLICY;
   }
 
+  /**
+   * Retry policy that uses up to about an hour of total wait time. Note that this is just the total waiting time
+   * between attempts. It does not include the time that each attempt takes to execute.
+   */
+  public static StandardRetryPolicy aboutAnHour()
```

Comment:
Nit: Would `StandardRetryPolicy.retryUptoAnHour()` or `StandardRetryPolicy.retryForAnHour()` communicate the intent better?

I initially got confused and thought "why would a policy retry _after_ an hour". 😅 

The "about" part could be left out as the approximate nature of the backoffs is probably a given. Although, I don't feel strongly about it.

Reply:
Hmm, to me they seem similar, so I'm inclined to merge the patch rather than have CI run again 🙂

Saving the planet by doing fewer CI runs!

Reply:
Haha, works for me. 🌲